### ISSUE TITLE
[FBref]  Error while scraping Spain comps using read_team_match_stats

### ISSUE THREAD

**Issue by Szymok** [probberechts/soccerdata#466]
Hello everyone,

I want to use a read_team_match_stats() to scrape team match stats, but the scraping is ending with error:
Internal Server Error for url:                   
https://fbref.com/en/squads/8d6fd021/matchlogs/all_comps/shooting.

![image](https://github.com/probberechts/soccerdata/assets/37596854/a8f3a697-4b43-49c6-9924-4783b517794d)

Correct url:
https://fbref.com/en/squads/8d6fd021/2023-2024/matchlogs/all_comps/shooting

So i think there is a problem with adding a seasong during building an url.

**Comment by probberechts** [comment]
This seems to work fine for me. It looks like this was fixed in #384. Are you sure you have the latest version?

```py
import soccerdata as sd
fbref = sd.FBref("ESP-La Liga", 2324, no_cache=True)
fbref.read_team_match_stats('shooting')
```

**Comment by Szymok** [comment]
Yes you're right, my fault, sorry
